qds_sdk/pipelines.py

```diff
@@ -339,23 +339,70 @@ def create(cls, pipeline_name, create_type, **kwargs):
 
         Args:
             pipeline_name: Name to be given.
-            create_type: 1->Assisted, 2->Code, 3->Jar
+            create_type: 1->Assisted, 2->Jar, 3->Code
             **kwargs: keyword arguments specific to create type
 
         Returns:
             response
         """
         conn = Qubole.agent()
-        data = {"data": {
-            "attributes":
-                {"name": pipeline_name, "status": "DRAFT",
-                 "create_type": create_type},
-            "type": "pipelines"}
-        }
-        url = Pipelines.rest_entity_path + "?mode=wizard"
+        url = Pipelines.rest_entity_path
+        if create_type is None:
+            raise ParseError("Provide create_type for Pipeline.", None)
+        if not kwargs or create_type == 1:
+            data = {
+                "data": {
+                        "attributes": {
+                            "name": pipeline_name,
+                            "status": "DRAFT",
+                            "create_type": create_type
+                        },
+                        "type": "pipeline"
+                    }
+            }
+            url = url + "?mode=wizard"
+        else:
+            data = {
+                "data": {
```

Comment:
can we store them in some constants? just makes it more readable here

Reply:
I didn't get you. Can you please explain with an example ?

Reply:
the JSON part can be sent to some constant variables, otherwise, it makes a bit more complicated here to read.

Reply:
oh didn't see kwargs are being used in these blocks, it's fine. Let's just add `kawargs.get` should be fine